benefits/templates/400.html

```diff
@@ -1 +1,19 @@
-{% extends "core/page.html" %}
+{% extends "core/base.html" %}
+
+{% block main_content %}
+  <div class="container">
+    {% include "core/includes/icon-title.html" with title=page.headline icon=page.icon %}
+
+    <div class="row justify-content-center">
+      <div class="col-lg-8 pt-4">
```

Comment:
I also noticed in b02d5a3 and some other commits like f00dc3121645f624757535f2a09de2021dd90258, the same change needed to be made across `400.html`, `404.html`, and `500.html`.

I see where you're coming from with
> In the future, the 400, 404, 500 pages could all inherit from a the same template. I'm think we should wait for this level of refactor until Design finishes redesigning these pages. https://github.com/cal-itp/benefits/issues/978

If the pages are just exact copies of each other though, I think it would be really easy to make an `error_page.html` template with that code and have `200_user_error.html`, `400.html`, `404.html`, `500.html`, and `unverified.html` extend that. That's my two cents at least, but I'll leave it up to you.

Reply:
Thanks for the feedback! My preference at the time being would be to prioritize #1054 first, which would allow us to then clean up the hacky button-filtering in `unverified` and `retry` templates, refactor the `agency-links` template to work for both `help` and `unverified`/`retry`, and then template-ize all the pages. 

I noticed the 200 page html was somehow _not_ formatted with 2 spaces so I also just made a commit to make it formatted the same as the others. I don't know how/why the linter/formatter allows this.